src/index.js

```diff
@@ -31,14 +31,24 @@ export default class Channel {
    * Add listener for event
    * @param {Function} callback
    * @param {Object} [context]
+   * @param {Boolean} [once]
    */
-  addListener (callback, context) {
+  addListener (callback, context, once) {
     this._ensureFunction(callback);
-    this._listeners.push({callback, context});
+    this._listeners.push({callback, context, once});
```

Comment:
I suggest not to change public api allowing to make the same thing in several ways.
Lets make internal `_pushListener` method that will be called by both `addListener ` and `addOnceListener`